src/Samples/Sample.AspNetCore/Controllers/CheckOutController.cs

```diff
@@ -244,15 +287,40 @@ public async Task<JsonResult> GetPaymentJsSource(PaymentInstrument instrument)
                 case PaymentInstrument.Swish:
                     var swishPayment = await CreateSwishPayment();
                     return new JsonResult(swishPayment.Operations.ViewSales.Href);
+
+                case PaymentInstrument.Trustly:
+                    var trustlyPayment = await CreateTrustlyPayment();
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)